### ISSUE TITLE
Data Entry page not allowing controlled vocabulary lists or expandable fields with subsuquent records following save

### ISSUE THREAD

**Issue by AJLinn** [ArctosDB/arctos#7534]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Describe the bug**
I successfully saved a new record and started on the second, nearly identical object (a blanket) but when I went to the fields I have set to "show" the pick lists for the attributes culture of origin and culture of use don't show up. All of the fields are simple white boxes and not able to expand in order to enter a long description in that attribute.

**Screenshots**
<img width="1468" alt="Screenshot 2024-03-13 at 12 45 52 PM" src="https://github.com/ArctosDB/arctos/assets/17605945/18539d5f-3eee-454b-89af-54ba57e0b47d">

**Desktop (please complete the following information):**
 - OS: Mac OS 14.2.1
 - Browser: Firefox
 - Version: 123.0.1 (64 bit)

**Priority**
High - I have to load incomplete records in order to finish the data entry work!

**Comment by dustymc** [comment]
I made a couple records in production, please be careful that they don't get loaded.

Your javascript console/log would be most useful - I grabbed one of your records and your profile and I can't break it, it's something in your network, browser, data, ???????????????????????? I'll keep looking, any clues would be most useful.


<img width="916" alt="Screenshot 2024-03-13 at 14 04 00" src="https://github.com/ArctosDB/arctos/assets/5720791/35715666-fe4d-47e3-9aaa-d7c228796224">

**Comment by AJLinn** [comment]
> Your javascript console/log would be most useful - I grabbed one of your records and your profile and I can't break it, it's something in your network, browser, data, ???????????????????????? I'll keep looking, any clues would be most useful.

One thing that did happen is I got kicked off Arctos (timed out) between saving the first one and then I logged back in and continued with my record creation. Maybe that's what did it? I now see that if I go in to edit from the browse/edit page I get the regular fields with dropdowns.
<img width="1468" alt="Screenshot 2024-03-13 at 1 55 08 PM" src="https://github.com/ArctosDB/arctos/assets/17605945/0b57097e-12ad-4bdd-b67c-67cbc4821981">

Should I go ahead and edit and load from there? I can delete the second record plus the fakefakerton one from the loader.
